integration_test/ops_agent_test.go

```diff
@@ -199,6 +199,106 @@ func setupOpsAgent(ctx context.Context, logger *logging.DirectoryLogger, vm *gce
 	return nil
 }
 
+func TestParseMultilineFile(t *testing.T) {
```

Comment:
@sophieyfang This test failed consistently across all platforms in the [Kokoro run](http://sponge/ac35acc3-7d30-42b4-92cb-03370321f1fa) for the final commit (https://github.com/GoogleCloudPlatform/ops-agent/commit/b98bacb4060f0b27f6fa13c1e77488e0efa1b3ad), and broke the nightly build on Debian Bullseye. Can you please investigate?

Reply:
I just checked the nightly job of `cloud-ops-agent_20220225_01`. Only debian11 failed at e2e test stage, it didn't fail on build stage. All the rest of os survived through e2e tests.

Reply:
I am investigating on why it's repeatedly happening in bullseye.

The cause is `ops_agent_test.go:294: QueryLog() failed: files_1 not found, exhausted retries`. i have a rough idea

Reply:
Reproduced and raised a bug in fluentbit side: https://github.com/fluent/fluent-bit/issues/4944

Reply:
I believe the same issue will happen on debian11 + mysql which uses tail multiline parsers...

Reply:
Just verified elastic search test also failed on debian11 with fluentbit 1.8.12.
Below is the testing config..
```
sophieyfang_google_com@debian11:~$ cat /etc/google-cloud-ops-agent/config.yaml 
# Copyright 2020 Google LLC
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#      http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

# <== Enter custom agent configurations in this file.
# See https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration
# for more details.
metrics:
  receivers:
    elasticsearch:
      type: elasticsearch
  service:
    pipelines:
      elasticsearch:
        receivers:
          - elasticsearch

logging:
  receivers:
    elasticsearch_json:
      type: elasticsearch_json
    elasticsearch_gc:
      type: elasticsearch_gc
  service:
    pipelines:
      elasticsearch:
        receivers:
          - elasticsearch_json
          - elasticsearch_gc

```